util/VarText.h

```diff
@@ -43,6 +43,7 @@ class FO_COMMON_API VarText {
     const std::string&          GetText() const;        //!< Returns text generated after substituting all variables.
     bool                        Validate() const;       //!< Does text generation succeed without any errors occurring?
     const std::string&          GetTemplateString() const   { return m_template_string; }
+    bool                        DoStringtableLookup() const { return m_stringtable_lookup_flag; }
```

Comment:
The member variable is preexisting in VarText. with the current name m_stringtable_lookup.  I will change this method name to GetStringtableLookupFlag.   This is at the VarText level, which doesn't have the concept of label, but SitrepPanel uses it as the flag for both.